hw/ip/keymgr/dv/keymgr_sim_cfg.hjson

```diff
@@ -38,7 +38,7 @@
                 ]
 
   // Add additional tops for simulation.
-  sim_tops: ["keymgr_bind", "sec_cm_bind"]
+  sim_tops: ["keymgr_bind", "sec_cm_prim_count_bind", "sec_cm_prim_sparse_fsm_flop_bind"]
```

Comment:
If we forget to do this, it will just show up as assertion coverage hole, right? Or will the compile fail?

Reply:
there will be an uncovered assertion coverage.